test/suites/integration/iamauthpolicy_test.go

```diff
@@ -21,25 +27,22 @@ import (
 
 var _ = Describe("IAM Auth Policy", Ordered, func() {
 
-	const Policy = `
-{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Effect": "Allow",
-      "Principal": "*",
-      "Action": "vpc-lattice-svcs:Invoke",
-      "Resource": "*"
-    }
-  ]
-}`
-
-	var ctx = context.Background()
-
-	newPolicyWithGroup := func(name, trGroup, trKind, trName string) anv1alpha1.IAMAuthPolicy {
-		p := anv1alpha1.IAMAuthPolicy{
+	const (
+		AllowAllInvoke = `{"Version":"2012-10-17","Statement":[{"Effect":"Allow","Principal":"*","Action":"vpc-lattice-svcs:Invoke","Resource":"*"}]}`
```

Comment:
Does policy with space, \n, \t has some issues? that make me remind one thing, should we delete \n \t for the policy string in the controller code?

Reply:
Spaces and new lines does not, empty ones should be a problem. This compact format is to make assertion easier, because when I GetAuthPolicy from lattice it returns it compact format, not beautified.